extensions/firebase-import-export/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "firebase-import-export",
+  "title": "Manage Firebase Firestore Collections",
+  "description": "Extension to export and import Firestore Collections (backup and restore). Including using Spark (free) subscription.",
+  "icon": "extension_icon.png",
+  "author": "mguyard",
+  "categories": [
+    "Data",
+    "Productivity",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "firebase-export",
+      "title": "Export Firebase",
+      "subtitle": "Export Collection(s)",
+      "description": "Export Firebase Collections",
+      "mode": "no-view"
+    },
+    {
+      "name": "firebase-import",
+      "title": "Import Firebase Collection(s)",
+      "subtitle": "Import Collections from a previous exported file",
+      "description": "Import Firebase Collection(s)",
+      "mode": "no-view"
+    }
+  ],
+  "preferences": [
+    {
+      "required": true,
+      "name": "firebaseAuth",
+      "title": "Firebase Authentication File",
+      "description": "You can Generate New Private Key from Project Settings from Firebase Console",
+      "type": "file"
+    },
+    {
+      "required": true,
+      "name": "exportFileStorage",
+      "title": "Storage location for export",
+      "description": "Where do you want to store export file",
+      "type": "directory"
+    },
+    {
+      "required": false,
+      "name": "collectionList",
+      "title": "Collections to export",
+      "description": "List of collections to export (separated by comma)",
+      "type": "textfield",
+      "placeholder": "collection1,collection2,collection3",
+      "default": ""
+    },
+    {
+      "required": false,
+      "name": "fileByCollection",
+      "label": "One file for each collection",
+      "description": "Each collection will be store in a separated file",
+      "type": "checkbox",
+      "default": false
+    },
+    {
+      "required": false,
+      "name": "limitDocumentExport",
+      "title": "Limit of exported documents",
+      "description": "limit number of documents when exporting (default: no limit)",
+      "type": "textfield",
+      "placeholder": "0"
+    }
```

Comment:
All of these preferences could be moved into the `Export Firebase` command's preferences to have cleaner root preferences.